xbmc/dialogs/GUIDialogSelect.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include "GUIDialogBoxBase.h"
+#include "video/VideoThumbLoader.h"
```

Comment:
This doesn't look good :/
Who is the consumer of this? Some window for versions that extend dialog select or just a raw dialog select?
I guess this should be moved elsewhere